gcp/modules/couchdb/main.tf

```diff
@@ -150,6 +152,11 @@ resource "null_resource" "couchdb_destroy_pvcs" {
         for PVC in $(kubectl get pvc --namespace ${var.release_namespace} -o json | jq --raw-output '.items[] | select(.metadata.name | startswith("database-storage-couchdb")) | .metadata.name'); do
           timeout -t 600 kubectl --namespace ${var.release_namespace} delete --ignore-not-found --grace-period=300 pvc $PVC
         done
+        # Some times GCP disks persist even after PVC destruction, removing them with gcloud to not leave stale resources behind
+        # More info: https://issues.gpii.net/browse/GPII-3743
+        for DISK in $(gcloud compute disks list --filter description:couchdb --format json | jq -r -c .[].name); do
```

Comment:
Quoting the `.[].name` bit would make me feel easier, as well as the output of the whole `$()`, otherwise this might be expanded by shell in case of some unexpected output.

Reply:
Nice catch!